src/audio.rs

```diff
@@ -0,0 +1,37 @@
+use bevy::prelude::*;
+
+/// An organizational marker component that should be added to a spawned [`AudioBundle`] if it is in the
+/// general "music" category (ex: global background music, soundtrack, etc).
+///
+/// This can then be used to query for and operate on sounds in that category. For example:
+///
+/// ```
```

Comment:
Are code blocks in doc quotes considered `rust` by default?

Reply:
Yep

Reply:
Ok cool thanks :)